crawlerr/services/memory_manager.py

```
@@ -0,0 +1,290 @@
+        # Just run garbage collection
+        collected = gc.collect()
+        if collected > 0:
+            logger.debug(f"Light cleanup: garbage collection freed {collected} objects")
+
+    def get_memory_info(self) -> dict[str, Any]:
+        """Get current memory information."""
+        if not self.available:
+            return {"available": False, "error": "psutil not available"}
+
+        # Force fresh check
+        asyncio.create_task(self.check_memory_pressure(force_check=True))
```

Ruff (0.12.2)

Store a reference to the return value of `asyncio.create_task`

(RUF006)